site/app/templates/grading/electronic/Details.twig

```diff
@@ -42,7 +42,7 @@
                 </a>
                 {% endif %}
                 <a class="btn btn-default"
-                   href="{{ order_toggle_url }}">
+                   href="" onclick="changeSortOrder()">
```

Comment:
It would be good to change this to a `<button>`, as per [this](https://github.com/Submitty/Submitty/issues/7819) issue since it no longer acts as a link.

Reply:
Interesting.  Given that it messes up the styling, I'd say it's ok to leave as-is.  You can make another PR to style the buttons properly if you want.  If you don't want to do that, I'd at least comment on https://github.com/Submitty/Submitty/issues/7819 and state that they styling is different (and why, if possible) so we can track it in the future if someone wants to work on it.